src/components/PrimaryNav/PrimaryNav.module.css

```diff
@@ -36,6 +41,18 @@
   @mixin TextLinkInverted;
   text-decoration: none;
   white-space: nowrap;
+  padding: 0.5rem;
+  border-radius: var(--eds-theme-border-radius);
+  transition: background var(--eds-anim-fade-quick) var(--eds-anim-ease);
+
+  &:hover,
+  &:focus {
+    background: var(--eds-color-brand-grape-800);
```

Comment:
```suggestion
    background: var(--eds-theme-color-text-link-strong-hover);
```

Reply:
Courtney and I were talking about how to handle these. text-link refers to actual text. Because this is technically a background, we may want to use the button-icon token for this. One of those weird background hover examples.